src/forwarder.ts

```diff
@@ -265,21 +266,21 @@ export async function describeSubscriptionFilters(aws: Aws, logGroupName: string
 }
 
 // Helper functions to validate we have a particular log group and if we should subscribe to it
-function validateRestApiSubscription(resource: any, subscribe: boolean, extendedProvider: any) {
+function validateRestApiSubscription(resource: any, subscribe: boolean, extendedProvider: any): boolean {
```

Comment:
Right now the return type suggested by IDE is `any`, probably because `resource` is `any`, but I believe the return type is intended to be `boolean`.